src/pykeen/models/predict.py

```diff
@@ -30,7 +30,7 @@ def get_head_prediction_df(
     relation_label: str,
     tail_label: str,
     *,
-    triples_factory: TriplesFactory,
+    triples_factory: CoreTriplesFactory,
```

Comment:
this probably isn't correct - these do rely on the labels

Reply:
fixed in 67e7b85